charts/postgres/values.yaml

```diff
@@ -0,0 +1,74 @@
+replicaCount: 1
+
+image:
+  repository: postgres
+  pullPolicy: IfNotPresent
+  tag: "16"
```

Comment:
We should note here that a valid tag must be specified that corresponds to the desired Postgres version we want to use.